src/i18n/en/nav.ts

```diff
@@ -12,72 +12,83 @@ export default [
 	{ text: 'Getting Started', slug: 'getting-started', key: 'getting-started' },
 	{ text: 'Installation', slug: 'install/auto', key: 'install' },
 	{ text: 'Editor Setup', slug: 'editor-setup', key: 'editor-setup' },
+	{ text: 'The Astro CLI', slug: 'reference/cli-reference', key: 'reference/cli-reference' },
 	{ text: 'Upgrade to v2', slug: 'guides/upgrade-to/v2', key: 'guides/upgrade-to/v2' },
-	{ text: 'Migrate to Astro', slug: 'guides/migrate-to-astro', key: 'guides/migrate-to-astro' },
-
-	{ text: 'Tutorials', header: true, type: 'learn', key: 'tutorials' },
-	{ text: 'Build a Blog', slug: 'tutorial/0-introduction', key: 'blog-tutorial' },
 
 	{ text: 'Core Concepts', header: true, type: 'learn', key: 'coreConcepts' },
 	{ text: 'Why Astro', slug: 'concepts/why-astro', key: 'concepts/why-astro' },
 	{ text: 'MPA vs. SPA', slug: 'concepts/mpa-vs-spa', key: 'concepts/mpa-vs-spa' },
 	{ text: 'Astro Islands', slug: 'concepts/islands', key: 'concepts/islands' },
 
+	{ text: 'Tutorials', header: true, type: 'learn', key: 'tutorials' },
+	{ text: 'Build a Blog', slug: 'tutorial/0-introduction', key: 'blog-tutorial' },
+	// { text: 'Thinking with Islands', slug: 'tutorial/0-introduction', key: 'island-tutorial' },
+
 	{ text: 'Basics', header: true, type: 'learn', key: 'basics' },
```

Comment:
I definitely agree with what you removed. To evaluate what remains, we need to flesh out the vision for Basics. What determines what you "need to know"? You can build a basic site without understanding anything but file-based routing, and certainly without understanding Markdown to the extent it's taught in the Markdown guide. `Pages` already covers the "basics" of both routing and Markdown, so I'd move both `Routing` and `Markdown & MDX` to `Guides`.

Reply:
Ooh, spicy!  :hot_pepper: 

Agreed, Routing was one I wasn't *sure* about, but I could see, until it's re-written, that MD/MDX goes in guides.